src/middlewared/middlewared/plugins/boot.py

```diff
@@ -203,3 +204,25 @@ class BootService(Service):
         """
         subjob = await self.middleware.call('zfs.pool.scrub', 'freenas-boot')
         return await job.wrap(subjob)
+
+    @accepts(
+        Int('interval', validators=[Range(min=1)])
+    )
+    async def scrub_interval(self, interval):
```

Comment:
If one is get_scrub_interval it only makes sense this one is called set_scrub_interval

Reply:
agreed